www/main/overlay/store/OverlayState.js

```diff
@@ -0,0 +1,34 @@
+import { createStore, action } from 'easy-peasy';
+import GlobalState from '../../common/store/GlobalState';
+import convertToCamelCase from '../../common/utils/convert-to-camel-case';
+
+global.platform = require('../../desktop_scripts');
+
+const createOverlayActions = () => {
+  const overlayActions = {};
+  Object.keys(GlobalState.getState().baniOverlay).forEach(stateVarName => {
+    const stateActionName = `set${convertToCamelCase(stateVarName, true)}`;
+    overlayActions[stateActionName] = action((state, payload) => {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
can you avoid it?

Reply:
I don't there's a clean way to avoid this. Fixing this would actually make the code a bit ugly.
This is the preffered method of updating easy peasy state, I can look up some doc link to be completely sure.

Reply:
See if you can, if not, why not just update the eslint config to allow for this?

Reply:
I think this is the only place where we would like to allow this , but on other places we would like to prevent param-reassign.